application/domain/user/auth.py

```diff
@@ -76,28 +75,22 @@ async def auth_token(db: AsyncSession, refresh_token: str) -> AuthUserResponse:
         user_id = payload.get("sub")
         user = await get_user_by_id(db, int(user_id))
         if not user.check_refresh_token(refresh_token):
-            raise HTTPException(
+            raise ErrorHttpException(
                 status_code=status.HTTP_401_UNAUTHORIZED,
-                detail={
-                    "error": "invalid_token",
-                    "error_description": "不正なリフレッシュトークンです",
-                },
+                error="invalid_token",
+                error_description="不正なリフレッシュトークンです。",
             )
         if user_id is None:
-            raise HTTPException(
+            raise ErrorHttpException(
                 status_code=status.HTTP_401_UNAUTHORIZED,
-                detail={
-                    "error": "unknown_user",
-                    "error_description": "不明なユーザーです",
-                },
+                error="invalid_token",
+                error_description="不明なユーザーです。",
```

Comment:
この人も `HTTP_400_BAD_REQUEST` が良さそう 🤔 
![image](https://github.com/takuron1996/mobile_reader_novel_back/assets/16306537/61c60abf-b435-4cce-ba37-538cfcb17f19)
> The provided token is malformed or otherwise not valid.	
> 提供されたトークンは不正な形式であるか、またはその他の点で有効ではありません。

Reply:
https://github.com/takuron1996/mobile_reader_novel_back/pull/3#discussion_r1554809396
と同様です。

Reply:
https://github.com/takuron1996/mobile_reader_novel_back/pull/3#discussion_r1554811078

ココで解決